.github/workflows/sycl-linux-run-tests.yml

```
@@ -142,6 +142,7 @@ on:
         options:
           - e2e
           - cts
+          - benchmark
```

```suggestion
          - benchmark
```
NIT: Would it be better to rename `benchmark` to `compute-benchmark`? The term `benchmark` is ambiguous

---

@uditagarwal97 I've changed it back to compute-benchmark. I had originally chosen benchmark over compute-benchmarks to account for the possibility of more benchmarks in the future (e.g. the UR benchmarking workflow runs multiple benchmarks)